src/lib/generate-top-level-sub-nav-items.ts

```diff
@@ -43,7 +43,6 @@ export const generateTopLevelSubNavItems = () => {
 		{ divider: true },
 		{ heading: 'Products' },
 		...betaProductItems,
-		{ divider: true },
```

Comment:
Subtask: [[bug] Two HRs on bottom of mobile nav](https://app.asana.com/0/1202097197789424/1203075873125158/f)

Could use extra testing please!

Reply:
Did some testing, and its looking good to me! I looked at docs, tutorials views at various sizes.

Reply:
@kendallstrautman thank you!